src/router.ts

```diff
@@ -1,8 +1,12 @@
 import { createWebHistory, createRouter } from 'vue-router';
 import LandingView from '@/views/LandingView.vue';
+import NotFoundView from '@/views/NotFoundView.vue';
 import type { Component } from 'vue';
 
-const routes = [{ path: '/', component: LandingView as Component }];
+const routes = [
+	{ path: '/', component: LandingView as Component },
```

Comment:
The `as Component` lines raised an eyebrow for me since that is a typecast -- we don't know it's a component, it could be anything.

Do you get a type error if you omit `as Component`?  When I make the change locally it seems fine.